### DISCUSSION TITLE
Which Kernel is this based on? Curious Synology user

### DISCUSSION THREAD

**Discussion by Hecatonchires667** [advplyr/audiobookshelf#3420] [Q&A]
Hi everyone! I´m using Synology with OS 7.1.1 which is based on Ubuntu 18 if i´m not mistaken. Usually everything above Linux 5.x Kernel leads to a crash of the whole system.

Is there any information regarding which image uses which kernel?

I´m not to familar with lx or Github. Unfortunately i´m not a developer.

Will give the latest buid a shot nonetheless. ;)

Kind regards

Marc

**Comment by advplyr** [comment]
Hey,
I'm not sure if this answers your question but the docker images use the latest version of alpine linux.
Currently:
```bash
/ # cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.20.3
PRETTY_NAME="Alpine Linux v3.20"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```
